<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZhengZhiYinSu</class>
 <widget class="QWidget" name="ZhengZhiYinSu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>整治因素</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>坡度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="PDComBoBox_0_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>地质灾害</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DZZGComBoBox_0_3"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>村庄区位条件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="QWTJComBoBox_1_0"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>村庄分布密度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CZFBMDComBoBox_1_1"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>乡名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxXiang"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>村名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCun"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>村庄空间形态</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="16" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>村庄空心化程度</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="21" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>整治对策</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>村庄发展等级</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="15" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>村庄基础条件</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>水文</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SWComBoBox_0_0">
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>土地覆被</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TDFBComBoBox_0_1"/>
     </item>
    </layout>
   </item>
   <item row="19" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>空废宗地聚集度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="KFZDJJDComBoBox_3_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>空置率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="KZLComBoBox_3_3"/>
     </item>
    </layout>
   </item>
   <item row="24" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="text">
        <string>下一条</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="18" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>常住人口比重</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CZRKBZComBoBox_3_0"/>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>废弃率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="FQLComBoBox_3_1"/>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>非农产业比重</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="FNCYBZComBoBox_2_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="0">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="23" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string>对策1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>对策2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>对策3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>对策4</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>基础条件</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>空间形态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>发展等级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>空心化程度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>综合意见</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="22" column="0">
    <spacer name="verticalSpacer_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>基础设施</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="JCSSComBoBox_2_0"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>公服设施</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="GFSSComBoBox_2_1"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
